totalRP3/Modules/Register/Characters/RegisterUIRelations.xml

```diff
@@ -0,0 +1,51 @@
+<!--
+	Copyright The Total RP 3 Authors
+	SPDX-License-Identifier: Apache-2.0
+-->
+<Ui xmlns="http://www.blizzard.com/wow/ui/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.blizzard.com/wow/ui/
+https://raw.githubusercontent.com/Meorawr/wow-ui-schema/main/UI.xsd">
+
+
+	<Frame name="TRP3_ConfigurationRelationsFrame" virtual="true" inherits="BackdropTemplate">
+		<KeyValues>
+			<KeyValue key="backdropInfo" value="TRP3_BACKDROP_MIXED_DIALOG_TOOLTIP_200_24_5555" type="global"/>
+		</KeyValues>
+		<Size x="0" y="60"/>
+		<Layers>
+			<Layer level="OVERLAY">
+				<FontString name="$parentTitle" text="[title]" inherits="GameFontNormal" justifyH="LEFT">
```

Comment:
Remind me to go through this later and eliminate all these `name="..."` attributes and convert them to parent keys.

Reply:
go through this later and eliminate all these name="..." attributes and convert them to parent keys.

Reply:
> Remind me to go through this later and eliminate all these `name="..."` attributes and convert them to parent keys.

go through this later and eliminate all these `name="..."` attributes and convert them to parent keys.